_maps/icebox.json

```diff
@@ -46,6 +46,7 @@
 		}
 	],
 	"minetype": "none",
+	"box_suffix": "/icebox",
```

Comment:
I'm not confident I like this over an array of additional items. This is more flexible, I guess?

Reply:
Do this differently. Create a `global` job_changes entry and add the field `box` and enter the full path to the box obj.